### PR TITLE
Add the verification feature of word count for nickname input in signup form

### PR DESCRIPTION
**Pull request by DomaLu**

### CODE REVIEW

File: public/js/components/Shared/HintChecked.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'hint-checked': true,
+    'show': show,
+    'valid' : valid
+  })
+
+  return (
+    <p className={hintClass}>
+      <span>
+        {count}
+      </span>/20
```
Comment (joyfeel) [inline public/js/components/Shared/HintChecked.js]:
```
<span>
  {count}
</span>/20
跟
<span>
  {count}/20
</span>
之間有何差別呢?
```

Comment (DomaLu) [inline public/js/components/Shared/HintChecked.js]: <span>
  {count}
</span>/20

可在<span>節點設計動態count的style

Comment (joyfeel) [inline public/js/components/Shared/HintChecked.js]: 了解

**Comment by joyfeel** [comment]
done

**Pull request closed without merging**